Pipfile

```diff
@@ -51,6 +50,7 @@ pdf2image = "*"
 flower = "*"
 bleach = "*"
 zxing-cpp = {version = "*", platform_machine = "== 'x86_64'"}
+django-multiselectfield = "*"
```

Comment:
I know it's confusing, but if you run `pipenv upgrade <new package> that will only do the work of adding the new package, without changing all the other versions.  Not a clear workflow at all